### PR TITLE
Add Customer Support Rep role to Handbook

### PR DESCRIPTION
**Pull request by lukemorton**
As part of ensuring every role is documented in the Handbook I submit the Customer Support Rep role for review.

In particular looking to discuss the expectations for this role.

### CODE REVIEW

File: roles/customer_support_representative.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Customer Support Representative
+
+Our Business Development Reps are at the forefront of building relationships with markets, organisations and key stakeholders. Ability to execute and evolve a plan is key.
```
Comment (chrisblackburn) [inline roles/customer_support_representative.md]: BDRs

File: roles/customer_support_representative.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+Over time you will form detailed knowledge of the various applications we have created for customers, including their purpose, functionality, hosting, metrics and permissions. This is information you will use to make support more effective, resolving issues yourself if needed, and continuing to ensure ever more effective triaging where not.
+
+The role offers a great deal of variety. Each of our customers uses a unique piece software with unique integrations and business purposes. We know of few support roles where the issues you are called upon to triage and resolve are so varied.
+
+This role provides the opportunity to progress your career in multiple different areas, including shaping or even leading our application support function as it grows. You might also decide to explore a career more centred around customer relationships, account management or service delivery depending on where your skill set naturally falls in performing the role.
```
Comment (chrisblackburn) [inline roles/customer_support_representative.md]: "skill set falls" sounds kinda negative. Where your aspirations lie, perhaps?

File: roles/customer_support_representative.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+You will escalate high priority issues to our reliability engineers and add low priority requests to the roadmap to be addressed at a later date. We use Asana to stay organised, so you will add each request to our customer boards as you receive them.
+
+You will act as single point of contact to ensure customers are informed and updated on the status of their technical support enquiry. You will also lead fortnightly reviews with each customer, discussing what has been delivered in previous iterations and which priorities remain for the following two weeks.
+
+Over time you will form detailed knowledge of the various applications we have created for customers, including their purpose, functionality, hosting, metrics and permissions. This is information you will use to make support more effective, resolving issues yourself if needed, and continuing to ensure ever more effective triaging where not.
```
Comment (chrisblackburn) [inline roles/customer_support_representative.md]: I think there's something maybe even bigger in here about understanding customer goals, where we can help further our impact with their organisation etc.?

File: roles/customer_support_representative.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+## Salary
+
+This role has a salary of £20-30k depending on experience.
+
+
+## Expectations
+
+### To keep customer roadmaps up to date with fortnightly calls
+
+We expect our Customer Support Rep to be keeping customer roadmaps up to date with their priorities at least every fortnight with a call. Reliability Engineers need to know what the customers next highest priority and roadmaps should be reflective of this.
```
Comment (chrisblackburn) [inline roles/customer_support_representative.md]: customer's

Comment (chrisblackburn) [reply roles/customer_support_representative.md]: "what the customer's next highest priority IS, and roadmaps..."

File: roles/customer_support_representative.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+### To work with reliability engineers to estimate time ranges for features, improvements and bug fixes
+
+We expect our Customer Support Rep to ensure all items in customer roadmaps have time range estimates in them. It is a collective responsibility for the Live Services team to make sure these are up to date.
+
+### To ensure customer roadmaps combine business value and reduction of technical debt
+
+We expect our Customer Support Rep to ensure items in customer roadmaps contain a mix of features, improvements and bug fixes that provide business value, as well as items that reduce technical debt over time. Customers often prefer the former and so effort must be spent to explain the important of reducing technical debt.
```
Comment (chrisblackburn) [inline roles/customer_support_representative.md]: educate over explain?

Comment (chrisblackburn) [reply roles/customer_support_representative.md]: importance instead of important

File: roles/customer_support_representative.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+### To work with reliability engineers to schedule 28 hours of work per week per engineer
+
+We expect Live Services to be as profitable as possible given it's shared resourcing model. We have set the minimum number of scheduled hours for Reliability Engineers to 28 hours per week representing 4 days of 7 hours billed. We expect our Customer Support Rep to support Reliability Engineers by managing a fortnightly schedule.
```
Comment (chrisblackburn) [inline roles/customer_support_representative.md]: its

Comment (chrisblackburn) [reply roles/customer_support_representative.md]: support Reliability Engineers in achieving this expectation by managing..

**Comment by chrisblackburn** [comment]
A few other thoughts:

- Financial reporting
- Accountable for customer happiness. Expected to proactively escalate issues that are damaging our reputation with the customer
- Ensure team ceremonies such as planning, retrospectives, and standups happen

File: roles/customer_support_representative.md (hunk 1 above)
Comment (lukemorton) [reply roles/customer_support_representative.md]: Woops!

File: roles/customer_support_representative.md (hunk 2 above)
Comment (lukemorton) [reply roles/customer_support_representative.md]: Haha, from Talent Point but sure 👍

File: roles/as_code/lib/shared_expectations/live_services_expectations.rb [hunk 7]
```diff
@@ -0,0 +1,10 @@
+class LiveServicesExpectations < JobSpec::Role::Expectations
+  expected 'to provide an outstanding customer service',
+    'We expect our Live Services team to be a delightful people to talk to for our customers. Our conversations should be professional and friendly, we should small talk where possible, use a smiley face once in a while and generally be nice folk.'
```
Comment (chrisblackburn) [inline roles/as_code/lib/shared_expectations/live_services_expectations.rb]: interact with over talk? IDK if talk implies phone/Hangout, and actually, much of our CS interaction is written

File: roles/as_code/lib/shared_expectations/live_services_expectations.rb [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+
+  expected 'to be a responsible remote worker',
+    'We expect our Live Services team to communicate more than anyone else in the business if they choose to work fully remote. Remote working requires extra attention to be paid to communication channels like email and slack.'
```
Comment (chrisblackburn) [inline roles/as_code/lib/shared_expectations/live_services_expectations.rb]: such as over like

Comment (chrisblackburn) [reply roles/as_code/lib/shared_expectations/live_services_expectations.rb]: Slack over slack

File: roles/as_code/lib/shared_expectations/live_services_expectations.rb [hunk 9]
```diff
@@ -0,0 +1,10 @@
...
+
+  expected 'to effectively juggle commitments',
+    'We expect our Live Services team to be able to balance the priority of their work. Having an understanding of the severity of work along with expected time to fix is important and it should be used to help prioritise what to work on. Customers expectations need to be set, met and if they cannot be met, be reset early and often.'
```
Comment (chrisblackburn) [inline roles/as_code/lib/shared_expectations/live_services_expectations.rb]: Customer over Customers

Comment (chrisblackburn) [reply roles/as_code/lib/shared_expectations/live_services_expectations.rb]: "met and if they cannot be met" is mega janky

File: roles/customer_support_representative.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+
+Over time you will form detailed knowledge of the various applications we have created for customers, including their purpose, functionality, hosting, metrics and permissions. This is information you will use to make support more effective, resolving issues yourself if needed, and continuing to ensure ever more effective triaging where not.
+
+The role offers a great deal of variety. Each of our customers uses a unique piece software with unique integrations and business purposes. We know of few support roles where the issues you are called upon to triage and resolve are so varied.
+
+This role provides the opportunity to progress your career in multiple different areas, including shaping or even leading our application support function as it grows. You might also decide to explore a career more centred around customer relationships, account management or service delivery depending on where your aspirations lie.
```
Comment (chrisblackburn) [inline roles/customer_support_representative.md]: Live Services over application support

**Pull request merged**